### PR TITLE
IA-3047 Submission photo

### PR DESCRIPTION
**Pull request by beygorghor**
We should be able to see photo fields while consulting submission 

Related JIRA tickets : IA-3047
## Self proofreading checklist

- [x] Did I use eslint and black formatters
- [ ] Is my code clear enough and well documented
- [ ] Are my typescript files well typed
- [ ] New translations have been added or updated if new strings have been introduced in the frontend
- [ ] My migrations file are included
- [ ] Are there enough tests
- [ ] Documentation has been included (for new feature)

## Changes

- New PhotoField for photo field type
- use imagegallery to open image

## How to test

Use this form to create a new form and create a submission with enketo.
[data_for_health_facilitydonnees_formation_sanitaire_2024051701.xlsx](https://github.com/user-attachments/files/15952112/data_for_health_facilitydonnees_formation_sanitaire_2024051701.xlsx)

When done, you should see your picture in the form content

## Print screen / video

Screenshot 2024-06-21 at 16 08 37
Screenshot 2024-06-21 at 16 04 03

### CODE REVIEW

File: hat/assets/js/apps/Iaso/domains/instances/components/InstanceFileContentRich.js [hunk 1]
```diff
@@ -147,16 +150,83 @@ export default function InstanceFileContentRich({
...
+    showQuestionKey: true,
+    files: [],
+};
+
+PhotoField.propTypes = {
+    descriptor: PropTypes.object.isRequired,
+    data: PropTypes.object.isRequired,
+    showQuestionKey: PropTypes.bool,
+    files: PropTypes.arrayOf(PropTypes.string),
+};
```
Comment (quang-le) [inline hat/assets/js/apps/Iaso/domains/instances/components/InstanceFileContentRich.js:227]: Shouldn't this be in its own file?

File: hat/assets/js/apps/Iaso/domains/instances/components/InstanceFileContentRich.js
Comment (quang-le) [inline hat/assets/js/apps/Iaso/domains/instances/components/InstanceFileContentRich.js:1]: We should really split all the components in this file, but not blocking for now. We shhould do it when we rewrite instances/submissions

**Review by quang-le — APPROVED** [review]
LGTM

File: hat/assets/js/apps/Iaso/domains/instances/components/InstanceFileContentRich.js (hunk 1 above)
Comment (beygorghor) [reply hat/assets/js/apps/Iaso/domains/instances/components/InstanceFileContentRich.js:227]: yes then i have to refactor all other fields due to circular deps

File: hat/assets/js/apps/Iaso/domains/instances/components/InstanceFileContentRich.js
Comment (beygorghor) [reply hat/assets/js/apps/Iaso/domains/instances/components/InstanceFileContentRich.js:1]: yes i agree

**Pull request merged**